pkg/api/types.go

```diff
@@ -78,8 +79,8 @@ type PodSchedulingSpec struct {
 	VirtualCluster          VirtualClusterName `yaml:"virtualCluster"`
 	Priority                int32              `yaml:"priority"`
 	PinnedCellId            PinnedCellId       `yaml:"pinnedCellId"`
-	GpuType                 string             `yaml:"gpuType"`
-	GpuNumber               int32              `yaml:"gpuNumber"`
+	LeafCellType            string             `yaml:"leafCellType"`
+	LeafCellNumber          int32              `yaml:"leafCellNumber"`
```

Comment:
I remember we said we keep both `gpu` and `leafCell` in the annotation?

Reply:
see 9cf2a59